sdk/schemaregistry/azure-schemaregistry/setup.py

```diff
@@ -13,65 +13,61 @@
 
 # Change the PACKAGE_NAME only to change folder and different name
 PACKAGE_NAME = "azure-schemaregistry"
-PACKAGE_PPRINT_NAME = "Schema Registry"
+PACKAGE_PPRINT_NAME = "Azure Schema Registry"
```

Comment:
you can add a `package-pprint-name` property in tspconfig.yaml for the python emitter to configure this value if you need any changes

Reply:
for now, we're not letting the emitter overwrite the existing setup.py, b/c we have some extras_require that we don't want overwritten. But will update this in the setup.py!